### DISCUSSION TITLE
More designing of the UNESCO Recommendation section

### DISCUSSION THREAD

**Discussion by JesperKawuma** [Open-Science-Community-Saudi-Arabia/OSCSA_Website#170] [Ideas]
Hello @BatoolMM, I would like to add some box shadow to the content in UNESCO Recommendation section to make it's view a bit better. I also want to know the time when the meetings are to be held plus the platform to be used. Thank you.

**Comment by BatoolMM** [comment]
@JesperKawuma Feel free to open an issue and PR to address this!
